app/src/main/java/org/wikipedia/search/SearchResult.kt

```diff
@@ -22,4 +23,11 @@ data class SearchResult(val pageTitle: PageTitle,
 
     constructor(pageTitle: PageTitle, searchResultType: SearchResultType) :
             this(pageTitle, null, searchResultType)
+
+    val location: Location? get() {
+        return if (coordinates.isNullOrEmpty()) null else
+            Location("").also {
+                it.latitude = coordinates[0].lat; it.longitude = coordinates[0].lon
```

Comment:
Do we need to check the first item of the `coordinates` exist? 

And, can we have separate lines when set `latitude` and `longitude`

Reply:
It's checking `isNullOrEmpty`, so the first item will definitely exist.
(and yes re separate lines)